src/components/UiCardList/cardData.tsx

```
@@ -0,0 +1,59 @@
+import { Card } from '../../features/landing/types/card-item';
```

ESLint

[error] `../../features/landing/types/card-item` import should occur after import of `../../assets/svg/why-us/templates.svg`

(import/order)